make.py

```diff
@@ -78,6 +78,8 @@ def get_generator(compiler, cpu):
 				# VS2017 ARM/ARM64 support only works with cmake 3.13 and up and the architecture must be specified with
 				# the -A cmake switch
 				return 'Visual Studio 15 2017'
+		elif compiler == 'vs2019':
```

Comment:
Which cmake version is required for vs2019 support?

Reply:
3.14; I have revised readme.md to say that.